tests/suites/test_suite_ecdsa.function

```diff
@@ -460,3 +460,68 @@ exit:
     mbedtls_ecdsa_free( &ctx );
 }
 /* END_CASE */
+
+/* BEGIN_CASE */
+void ecdsa_verify( int grp_id, char * x, char * y, char * r, char * s, int len, int key_is_valid, int expected )
+{
+    uint8_t FUZZ_DATA[] = {
```

Comment:
what's the purpose of `FUZZ_DATA`? is it actually `dummy_data /* as used by the fuzzer that originally found the problem */`?

Reply:
Yes, this is the data the fuzzer came up with. It's actually sufficient to have less than this but I felt it was handy to have something exactly corresponding to the bug report

Reply:
is `FUZZ_DATA` a good name for this variable when someone comes back to look at this test in 6 months time?

Reply:
I think it's reasonably clear (it is also explained in the commit message)? In any case, following the PR number in the commit messge back to the github issue will give the background if needed